jwst/dark_current/dark_current_step.py

```diff
@@ -68,6 +75,36 @@ def process(self, input):
 
         return out_ramp
 
+    def set_average_dark_current(self, input_model, dark_model):
+        '''Take the three possible locations specifying
+        the average dark current and assign them to the
+        input model, in priority order:
+        1) Any value provided to the step parameter, either from
+        the user or a parameter reference file
+        2) The 2-D array stored in dark_model.average_dark_current
+        3) The scalar value stored in dark_model.meta.exposure.average_dark_current
+
+        Inputs
+        ------
+        input_model : stdatamodels.jwst.datamodels.RampModel
+            The input datamodel containing the 4-D ramp array
+        dark_model : Union[stdatamodels.jwst.datamodels.DarkModel,
+                           stdatamodels.jwst.datamodels.DarkMIRIModel]
```

Comment:
Sphinx is still choking on this line of the doc string. I think the indent level of line 92 needs to be fixed.